## This is the contents of an episode within the Episodic Memory
## Related pkg: eris
##
## Author: Ferdian Pratama

Header header
string label
int16 obj_count
int16 seq_count
int16[] pos_minx
int16[] pos_maxx
int16[] pos_miny
int16[] pos_maxy
#vi_sor/localColor[] obj_color # color descriptor do not use, just search the obj label
string[] obj_name #filename
string[] sequence